test/test_ops.py

```diff
@@ -270,7 +270,10 @@ def test_noncontiguous_samples(self, device, dtype, op):
             actual_grad = n_inp.grad if isinstance(n_inp, torch.Tensor) else n_inp[0].grad
 
             # TODO: FIXME: only validates grad on first tensor input
-            self.assertEqual(actual_grad, expected_grad)
+            if dtype == torch.complex64:
```

Comment:
This is not good. You should fix this by adding a decorator to modify the atol / rtol for this test from within the OpInfo.

Reply:
Ah that looks a nicer way of doing it. Thanks!